src/node-helpers.js

```diff
@@ -5,8 +5,12 @@ export const isGettextFuncCall = (names, node) => (
     : names.indexOf(node.callee.name) !== -1
 );
 
-export const isGettextComponent = (names, node) =>
-  names.indexOf(node.name.name) !== -1;
+export const isGettextComponent = (names, node) => {
+  if (!node || node.type !== 'JSXOpeningElement') {
```

Comment:
Could you add a comment here explaining why this check is done?

Reply:
I added a comment but feel free to modify if it's not appropriate :)